express/scripts/utils/autocomplete-api-v3.js

```diff
@@ -0,0 +1,81 @@
+import { getConfig } from '../utils.js';
+import { memoize, throttle, debounce } from './hofs.js';
```

Comment:
can you move this whole file out of utils? 
its only for the templates right? 
so maybe put it in a folder named template-x-utils/ or I don't know. But I don't want our utils folder packed with tons of crap again.